src/v2/Apps/Consign/Components/GetPriceEstimate.tsx

```diff
@@ -1,24 +1,114 @@
 import React from "react"
-import { Box, Flex, Text } from "@artsy/palette"
+import {
+  Box,
+  Flex,
+  Image,
+  Input,
+  Join,
+  MagnifyingGlassIcon,
+  Spacer,
+  Text,
+  breakpoints,
+} from "@artsy/palette"
+import { SectionContainer } from "./SectionContainer"
 
 export const GetPriceEstimate: React.FC = () => {
   return (
-    <Flex
-      flexDirection={["column", "row"]}
-      alignItems="center"
-      justifyContent="center"
-    >
-      <Box>
-        <Text variant="title">Get a Price Estimate</Text>
-        <Text variant="text">
-          Discover the potential of reselling an artwork you own.
-        </Text>
-      </Box>
-      <Box>
-        <Box>Artwork</Box>
-        <Box>Artwork</Box>
-        <Box>Artwork</Box>
-      </Box>
-    </Flex>
+    <SectionContainer background="black5">
+      <Flex
+        width="100%"
+        maxWidth={breakpoints.xl}
+        flexDirection={["column", "column", "column", "row"]}
```

Comment:
Note that i'm breaking into a vertical layout at the `md` breakpoint so that things aren't cramped